lib/commons/aria/attributes.js

```diff
@@ -45,70 +45,3 @@ aria.validateAttr = function(att) {
 	'use strict';
 	return !!aria.lookupTable.attributes[att];
 };
-
-/**
```

Comment:
I wish this file was moved in a separate commit so I could see if there were any changes to it side by side.... Lots of style changes mixed in here.

Reply:
I'm not sure how to do this.

Reply:
I thought maybe you could make the changes in the existing file in one commit and move/rename it in a separate commit. I don't think it's worth holding up this PR, just something to keep in mind moving forward.